ush/config_defaults.yaml

```diff
@@ -2427,6 +2475,45 @@ global:
 #----------------------------
 # verification (vx) parameters
 #-----------------------------
+verification_resources:
```

Comment:
I pulled this one completely out to it's own section because the verification tasks are just so many that I couldn't do it in this round.